File: tests/human/x/hs/match_full.hs
module Main where

classify :: Int -> String
classify n = case n of
  0 -> "zero"
  1 -> "one"
  _ -> "many"

main :: IO ()
main = do
  let x = 2
      label = case x of
                 1 -> "one"
                 2 -> "two"
                 3 -> "three"
                 _ -> "unknown"
  putStrLn label

  let day = "sun"
      mood = case day of
                "mon" -> "tired"
                "fri" -> "excited"
                "sun" -> "relaxed"
                _     -> "normal"
  putStrLn mood

  let ok = True
      status = if ok then "confirmed" else "denied"
  putStrLn status

  putStrLn (classify 0)

File: tests/human/x/python/match_full.py
x = 2
if x == 1:
    label = "one"
elif x == 2:
    label = "two"
elif x == 3:
    label = "three"
else:
    label = "unknown"
print(label)

day = "sun"
if day == "mon":
    mood = "tired"
elif day == "fri":
    mood = "excited"
elif day == "sun":
    mood = "relaxed"
else:
    mood = "normal"
print(mood)

ok = True
status = "confirmed" if ok else "denied"
print(status)

def classify(n: int) -> str:
    if n == 0:
        return "zero"
    elif n == 1:
        return "one"
    else:
        return "many"

print(classify(0))
print(classify(5))
